Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM miyurud/jasminegraph-prerequisites:20240101T095619
+FROM pre:latest
```

Comment:
Why do we change the base image here?

Reply:
it was mistakenly commited my locally used prerequisite image. now I edited it and push it.

Reply:
Ok